mopidy_local/commands.py

```diff
@@ -1,177 +1,172 @@
+        self.add_child("scan", ScanCommand())
+        self.add_child("clear", ClearCommand())
 
 
 class ClearCommand(commands.Command):
-    help = 'Clear local media files from the local library.'
+    help = "Clear local media files from the local library."
 
     def run(self, args, config):
         library = _get_library(args, config)
         if library is None:
             return 1
 
-        prompt = '\nAre you sure you want to clear the library? [y/N] '
+        prompt = "\nAre you sure you want to clear the library? [y/N] "
 
-        if compat.input(prompt).lower() != 'y':
-            print('Clearing library aborted.')
+        if input(prompt).lower() != "y":
+            print("Clearing library aborted.")
             return 0
 
         if library.clear():
-            print('Library successfully cleared.')
+            print("Library successfully cleared.")
             return 0
 
-        print('Unable to clear library.')
+        print("Unable to clear library.")
         return 1
 
 
 class ScanCommand(commands.Command):
-    help = 'Scan local media files and populate the local library.'
+    help = "Scan local media files and populate the local library."
 
     def __init__(self):
-        super(ScanCommand, self).__init__()
-        self.add_argument('--limit',
-                          action='store', type=int, dest='limit', default=None,
-                          help='Maximum number of tracks to scan')
-        self.add_argument('--force',
-                          action='store_true', dest='force', default=False,
-                          help='Force rescan of all media files')
+        super().__init__()
+        self.add_argument(
+            "--limit",
+            action="store",
+            type=int,
+            dest="limit",
+            default=None,
+            help="Maximum number of tracks to scan",
+        )
+        self.add_argument(
+            "--force",
+            action="store_true",
+            dest="force",
+            default=False,
+            help="Force rescan of all media files",
+        )
 
     def run(self, args, config):
-        media_dir = config['local']['media_dir']
-        scan_timeout = config['local']['scan_timeout']
-        flush_threshold = config['local']['scan_flush_threshold']
-        excluded_file_extensions = config['local']['excluded_file_extensions']
+        media_dir = config["local"]["media_dir"]
+        scan_timeout = config["local"]["scan_timeout"]
+        flush_threshold = config["local"]["scan_flush_threshold"]
+        excluded_file_extensions = config["local"]["excluded_file_extensions"]
         excluded_file_extensions = tuple(
-            bytes(file_ext.lower()) for file_ext in excluded_file_extensions)
+            file_ext.lower() for file_ext in excluded_file_extensions
+        )
 
         library = _get_library(args, config)
         if library is None:
             return 1
 
         file_mtimes, file_errors = path.find_mtimes(
```

Comment:
In Mopidy, `mopidy.internal.path.find_times()` is now a reexport of `mopidy.internal.mtimes.find_times()`. It is only used by `tests/audio/test_scan.py`. I think that we should either:

- conclude that the `mopidy.internal.mtimes` module belongs in Mopidy-Local and not Mopidy, or
- make `find_times()` a public method, maybe in the `mopidy.audio.scan` module?

@adamcik Any thoughts on the placement of the scanner and its supporting functions?

Reply:
Moved discussion to #19.